SitumWayfinding/Classes/Positioning/Navigation/InfoBarNavigationViewController.swift

```diff
@@ -121,3 +100,68 @@ extension SITNavigationProgress {
         }
     }
 }
+
+extension InfoBarNavigationViewController {
+    func prepareElements() {
+        if #available(iOS 13.0, *) {
+            if traitCollection.userInterfaceStyle == .light {
+                self.prepareLightOrDarkMode(tintColor: UIColor.black, colorText: UIColor.black)
+            } else {
+                self.prepareLightOrDarkMode(tintColor: UIColor.white, colorText: UIColor.white)
+            }
+        } else {
+            let cancelImage = UIImage(
+                named: "situm_navigation_cancel",
+                in: SitumMapsLibrary.bundle, compatibleWith: nil
+            )
+            
+            self.distanceRemainingImage.image = UIImage(
+                named: "situm_walk",
+                in: SitumMapsLibrary.bundle,
+                compatibleWith: nil
+            )
+            self.estimatedTimeImage.image = UIImage(
+                named: "situm_clock_time",
+                in: SitumMapsLibrary.bundle,
+                compatibleWith: nil
+            )
+            
+            self.cancelButton.setImage(cancelImage, for: .normal)
+            self.cancelButton.tintColor = .primary
+            self.timeRemainingLabel.textColor = .primary
+            self.distanceRemainingLabel.textColor = .primary
+            self.estimatedTimeLabel.textColor = .primary
+            self.separatorLabel.textColor = .primary
+        }
+        
+        self.timeRemainingLabel.font = .normalBold
+        self.distanceRemainingLabel.font = .small
+        self.estimatedTimeLabel.font = .small
+        self.separatorLabel.font = .normal
+        self.topSeparator.backgroundColor = .primaryDiminished
+        
+        
+    }
+    
+    @available(iOS 13.0, *)
+    func prepareLightOrDarkMode(tintColor: UIColor, colorText: UIColor) {
```

Comment:
Quedo que o códgio quedaría moito máis claro se fas tres métodos:

customizeUIForLightMode(){
applyStyleToUIElements(tintColor: UIColor.black, colorText: UIColor.black)
}

customUIForDarkMode(){
applyStyleToUIElements(tintColor: UIColor.white, colorText: UIColor.white
}

applyStyleToUIElements(tintColor: UIColor, colorText: UIColor) // que sería o teu prepareLightOrDarkMode